docs/handbook/time-off.md

```diff
@@ -0,0 +1,49 @@
+# Time off
+
+PostHog encourage its team to take time off to recharge.
+
+We have a flexible time off policy. Sometimes you need an extra day or two.
+
+We believe people need 20 days off a year to have meaningful time with their families plus a sprinkling of national holidays, to explore or just to relax.
+
+PostHog therefore offer unlimited time off, but with an expectation that you take at least 25 days off a year inclusive of national holidays.
+
+This is to make sure that people can take time off flexibly, whilst not feeling guilty about taking time off.
+
+The reason for this policy is that it's table-stakes for PostHog that we hire people we can trust to be responsible with their time off - enough that they can recharge, but not so much that it means we don't get any work done.
+
+## Permissionless time off
+
+You do not need to "clear" time off with your manager.
+
+We care about your results, not how long you work. Whilst no approval is needed, it shouldn't be at the expense of business getting done. For example, having the entire technical team off means we can't be responsive to community issues. Please coordinate with your team.
+
+When you pick a date(s) to have off, please enter them into the [shared vacation calendar](https://calendar.google.com/calendar?cid=cG9zdGhvZy5jb21fa3Rzb2hjMmZtaWFkbHF2ZWI0bmk0NG1tbzBAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ). Remember to set an out of office message on your email.
+
+## Longer breaks
+
+The same rules as above apply regardless of the vacation length.
+
+## When you should have time time off
+
+### You are sick
+
+If you are sick, you don't need to work and you will be paid. This is assuming you need a day or two off, just take them.
+
+If it was anticpated you'd be working in anything synchronous (ie calls with clients, standups), we'd appreciate a heads up. However, if you're really very unwell, please don't worry about work - the world won't end.
```

Comment:
This kinda feels like "you don't need to let anyone know if you're ill" which I don't agree with. I do feel people should inform their manager if they won't be working that day as otherwise there will be confusion during standup, asking for reviews etc.